docs/pages/admin-guides/management/operations/backup-restore.mdx

```diff
@@ -2,211 +2,118 @@
 title: Backup and Restore
 description: How to back up and restore your Teleport cluster state.
 ---
-This guide explains the components of your Teleport deployment that must be
-backed up and lays out our recommended approach for performing backups.
 
-(!docs/pages/includes/cloud/call-to-action.mdx!)
-
-## What you should back up
-
-### Teleport services
-<Tabs>
-<TabItem scope={["enterprise", "oss"]} label="Self-Hosted">
+Operators must have a plan in place to back up self-hosted Teleport clusters.
+While the Teleport Proxy Service and Teleport Agent services are stateless, you
+should ensure that you can restore their configuration files. The Teleport Auth
+Service manages state for the entire cluster, and it is critical that you can
+back up its data. This guide explains the components of a Teleport Auth Service
+deployment that must be backed up and lays out our recommended approach for
+performing backups. 
 
-Teleport's Proxy Service and Nodes are stateless. For these components, only
-the configuration file (`/etc/teleport.yaml` by default) should be backed up.
+(!docs/pages/includes/cloud/call-to-action.mdx!)
 
-The Auth Service is Teleport's brain, and depending on the backend should be
-backed up regularly.
+## Data to back up
 
-For example, a Teleport cluster running on AWS with DynamoDB must back up the
-following data:
+You must back up the following Auth Service data. The cluster state backend is a
+key-value store for dynamic resources, audit events, and other state data. The
+session recording backend is an Amazon S3-compatible object store:
 
-| What | Where ( Example AWS Customer ) |
+| What | Where |
 | - | - |
-| Local Users ( not SSO ) | DynamoDB |
-| Certificate Authorities | DynamoDB |
-| Trusted Clusters | DynamoDB |
-| Connectors: SSO | DynamoDB / File System |
-| RBAC | DynamoDB / File System |
-| teleport.yaml | File System |
-| teleport.service | File System |
-| license.pem | File System |
-| TLS key/certificate | File System / AWS Certificate Manager |
-| Audit log | DynamoDB |
-| Session recordings | S3 |
-
-For this customer, we would recommend using [AWS best practices](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/BackupRestore.html) for backing up DynamoDB. If DynamoDB is used for
-Teleport audit logs, logged events have a TTL of 1 year.
+| Local Users (not SSO) | Cluster state backend |
+| Certificate Authorities | Cluster state backend |
+| Dynamic resources ([more information below](#dynamic-resources)) | Cluster state backend |
+| teleport.yaml | File system |
+| teleport.service | File system |
+| license.pem | File system |
+| TLS key/certificate | File system or third-party service (e.g., AWS Certificate Manager)|
+| Audit log | Cluster state backend |
+| Session recordings | Session recording backend |
+
+## Backing up Teleport backends
+
+Your plan for backing up the Teleport cluster state and session recording
```

Comment:
The table below only covers the cluster state backends - it doesn't mention session recording backends (S3-compatible storage) at all.

Reply:
Adding a separate table for session recording backends